hadoop-hdds/docs/content/design/ofs.md

```diff
@@ -3,7 +3,7 @@ title: Implement new Ozone FileSystem scheme ofs://
 summary: A new schema structure for Hadoop compatible file system
 date: 2019-12-05
```

Comment:
can we update the date given the status changed below?

Reply:
no problem